app/views/shared/forms/_shared_ownership_form.html.erb

```diff
@@ -7,17 +7,22 @@
   <% group_error_class = error ? 'govuk-form-group--error' : '' %>
   <% input_error_class = error ? 'govuk-input--error' : '' %>
   <%= content_tag(:span, error, class: 'govuk-error-message', id: 'shared_ownership') if error %>
-  <%= govuk_fieldset_header(content_for(:page_title)) %>
-  <div class="govuk-form-group <%= group_error_class %>">
-    <div class="govuk-radios govuk-!-padding-top-2">
-      <%= render partial: 'shared/forms/shared_ownership_form/shared_ownership_item',
-                 collection: LegalAidApplication::SHARED_OWNERSHIP_REASONS, as: :reason,
-                 locals: { form: form,
-                           group_error_class: group_error_class,
-                           input_error_class: input_error_class } %>
 
+  <%= govuk_form_group show_error_if: @form.errors.present? do %>
+    <%= govuk_fieldset_header page_title %>
+
+    <div class="govuk-form-group <%= group_error_class %>">
```

Comment:
This `div` needs to be removed. The `govuk_form_group` helper inserts the content into the shared template `app/views/shared/forms/_form_group.html.erb`. If you look at that you should see that this already adds a "govuk-form-group" and handles the error class.